CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.2.0]
+### Changed
+* metadata tile script name from `generate_frame_tile.py` to `generate_metadata_tile.py`
+
```

Comment:
I'd add in the change to the input here as well (technically could be a breaking change?)

Reply:
done!

Reply:
Since we're pre 1.0.0 release, I'd prefer to not do a major version bump, even though it is a breaking change.